app-games/src/main/java/com/aptoide/android/aptoidegames/appview/AppViewScreen.kt

```diff
@@ -262,9 +267,67 @@ fun AppViewContent(
         navigate = navigate
       )
     }
+    if (app.isAppCoins) {
+      Row(
+        modifier = Modifier
+          .padding(top = 160.dp)
+          .align(Alignment.TopStart)
+      ) {
+        Image(
+          imageVector = getBonusIconLeft(
+            iconColor = Palette.Primary,
+            outlineColor = Palette.Black,
+            backgroundColor = Palette.Secondary
+          ),
+          contentDescription = "AppCoins Icon",
+          modifier = Modifier
+            .size(40.dp)
+            .graphicsLayer {
+              this.translationY = 6.dp.toPx()
+            }
+        )
+        Box(
```

Comment:
Maybe this should be a reusable composable component where we pass the text and the configurable style